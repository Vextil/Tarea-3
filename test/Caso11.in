# stress ListaString
# Prueba de orden de tiempo
TiempoLista
# Prueba de uso de memoria
MemoriaLista
Fin
