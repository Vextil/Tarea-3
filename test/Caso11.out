1># stress ListaString
2># Prueba de orden de tiempo
3>Fin Insertar. Fin Cantidad. Fin Remover. 
4># Prueba de uso de memoria
5>Fin Destruir. Fin Remover. 
6>Fin
